src/helpers/array-methods.jsx

```diff
@@ -0,0 +1,67 @@
+import {inventory} from "../constants/inventory.js";
+import React from "react";
+
+const tvTypes = inventory.map((tv) => {
+    return tv.type
+})
+
+const soldOut = inventory.filter((tv) => {
+    if (tv.originalStock === tv.sold) {
+        return tv
+    }
+})
```

Comment:
De filter functie verwacht een boolean terug te krijgen. De reden dat het werkt is omdat een object truthy is en `undefined` falsy. Desondanks zou ik altijd expliciet willen zijn:
```suggestion
const soldOut = inventory.filter((tv) => {
    return tv.originalStock === tv.sold;
})
```
Of nog korter:
```suggestion
const soldOut = inventory.filter((tv) => tv.originalStock === tv.sold);
```